webui/react/src/components/ResponsiveTable.tsx

```diff
@@ -19,11 +21,23 @@ const ResponsiveTable: ResponsiveTable = ({ scroll, ...props }) => {
     if (tables.length === 0) return;
 
     const rect = tables[0].getBoundingClientRect();
+
+    // ant table scrolling has an odd behaviour, if scroll.x is set to 'max-content' initially
+    // it will show the scroll bar, we need to set it to undefined the first time if scrolling
+    // is not needed, and max-content if we want to disable scrolling after it has been displayed
```

Comment:
**nit:** For multi-format comments, use the `/* comment... */` format.
```
/*
 * ant table scrolling has an odd behaviour. If scroll.x is set to 'max-content' initially
 * it will show the scroll bar. We need to set it to undefined the first time if scrolling
 * is not needed, and 'max-content' if we want to disable scrolling after it has been displayed.
 */
```